### ISSUE TITLE
Vu+ SAT>IP client support?

### ISSUE THREAD

**Issue by rezna11** [Barracuda09/SATPI#50]
Hi, I run SATPI on raspberry, Vu+ finds it but when I try to scan channels on a satellite, nothing happens. Is  scanning function supported by SATPI? Streaming a concrete channel in VLC works, but I need to scan the frequencies first, without that, there is no channel list...
Thanks

**Comment by Barracuda09** [comment]
Hi @rezna11 ,

Yes, scanning is supported. Could you give me a log of SatPI when you try to scan?

Thanks,

Marc

**Comment by Barracuda09** [comment]
Hi @rezna11 ,

You could please try it again with latest commit?

Thanks for giving it a try,

Marc

**Comment by rezna11** [comment]
well that's the problem, there is nothing in the log - the only thing is the discovery process - the Vu+ and Dreambox both find the SATPI server but when I try to tune in channels, nothing new is shown in the SATPI log. This worked with tvheadend, so I'd say the Vu+/Dreambox side is OK

**Comment by Barracuda09** [comment]
Could you try new commit

**Comment by Barracuda09** [comment]
Did you build a debug version?

**Comment by rezna11** [comment]
I did build a new version but not debug. I will try debug.

**Comment by rezna11** [comment]
ok my bad - the new build didn't run because I thought "sudo killall -9 satpi" would end satpi but no, it didn't so the ports were blocked... after rebooting raspberry, I was able to scan and stream but there was another problem, after a short while, when trying to scan other transponders, the log went crazy and it was impossible to scan new frequencies... here's part of the log:

`0  Sat Oct  8 18:28:02.0333 2016  7  Location: 
1  Sat Oct  8 18:28:02.0333 2016  7  CSeq: 3529
2  Sat Oct  8 18:28:02.0333 2016  7  cache-control: no-cache
3  Sat Oct  8 18:28:02.0333 2016  7  Content-Type: video/MP2T
4  Sat Oct  8 18:28:02.0333 2016  7  --- LINE END ---
5  Sat Oct  8 18:28:02.0492 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
6  Sat Oct  8 18:28:02.0492 2016  7  CSeq: 3530
7  Sat Oct  8 18:28:02.0492 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
8  Sat Oct  8 18:28:02.0492 2016  7  --- LINE END ---
9  Sat Oct  8 18:28:02.0499 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 2105829603
10  Sat Oct  8 18:28:02.0501 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
11  Sat Oct  8 18:28:02.0503 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
12  Sat Oct  8 18:28:02.0505 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
13  Sat Oct  8 18:28:02.0506 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
14  Sat Oct  8 18:28:02.0506 2016  7  Server: SatPI WebServer v0.1
15  Sat Oct  8 18:28:02.0506 2016  7  Location: 
16  Sat Oct  8 18:28:02.0506 2016  7  CSeq: 3530
17  Sat Oct  8 18:28:02.0506 2016  7  cache-control: no-cache
18  Sat Oct  8 18:28:02.0506 2016  7  Content-Type: video/MP2T
19  Sat Oct  8 18:28:02.0506 2016  7  --- LINE END ---
20  Sat Oct  8 18:28:02.0710 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
21  Sat Oct  8 18:28:02.0710 2016  7  CSeq: 3531
22  Sat Oct  8 18:28:02.0710 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
23  Sat Oct  8 18:28:02.0710 2016  7  --- LINE END ---
24  Sat Oct  8 18:28:02.0717 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1354574427
25  Sat Oct  8 18:28:02.0721 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
26  Sat Oct  8 18:28:02.0723 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
27  Sat Oct  8 18:28:02.0724 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
28  Sat Oct  8 18:28:02.0726 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
29  Sat Oct  8 18:28:02.0726 2016  7  Server: SatPI WebServer v0.1
30  Sat Oct  8 18:28:02.0726 2016  7  Location: 
31  Sat Oct  8 18:28:02.0726 2016  7  CSeq: 3531
32  Sat Oct  8 18:28:02.0726 2016  7  cache-control: no-cache
33  Sat Oct  8 18:28:02.0726 2016  7  Content-Type: video/MP2T
34  Sat Oct  8 18:28:02.0726 2016  7  --- LINE END ---
35  Sat Oct  8 18:28:02.0924 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
36  Sat Oct  8 18:28:02.0924 2016  7  CSeq: 3532
37  Sat Oct  8 18:28:02.0924 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
38  Sat Oct  8 18:28:02.0924 2016  7  --- LINE END ---
39  Sat Oct  8 18:28:02.0931 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1740892522
40  Sat Oct  8 18:28:02.0934 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
41  Sat Oct  8 18:28:02.0937 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
42  Sat Oct  8 18:28:02.0938 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
43  Sat Oct  8 18:28:02.0940 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
44  Sat Oct  8 18:28:02.0940 2016  7  Server: SatPI WebServer v0.1
45  Sat Oct  8 18:28:02.0940 2016  7  Location: 
46  Sat Oct  8 18:28:02.0940 2016  7  CSeq: 3532
47  Sat Oct  8 18:28:02.0940 2016  7  cache-control: no-cache
48  Sat Oct  8 18:28:02.0940 2016  7  Content-Type: video/MP2T
49  Sat Oct  8 18:28:02.0940 2016  7  --- LINE END ---
50  Sat Oct  8 18:28:02.1064 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
51  Sat Oct  8 18:28:02.1064 2016  7  CSeq: 3533
52  Sat Oct  8 18:28:02.1064 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
53  Sat Oct  8 18:28:02.1064 2016  7  --- LINE END ---
54  Sat Oct  8 18:28:02.1071 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1797702436
55  Sat Oct  8 18:28:02.1074 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
56  Sat Oct  8 18:28:02.1076 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
57  Sat Oct  8 18:28:02.1078 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
58  Sat Oct  8 18:28:02.1080 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
59  Sat Oct  8 18:28:02.1080 2016  7  Server: SatPI WebServer v0.1
60  Sat Oct  8 18:28:02.1080 2016  7  Location: 
61  Sat Oct  8 18:28:02.1080 2016  7  CSeq: 3533
62  Sat Oct  8 18:28:02.1080 2016  7  cache-control: no-cache
63  Sat Oct  8 18:28:02.1080 2016  7  Content-Type: video/MP2T
64  Sat Oct  8 18:28:02.1080 2016  7  --- LINE END ---
65  Sat Oct  8 18:28:02.1255 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
66  Sat Oct  8 18:28:02.1255 2016  7  CSeq: 3534
67  Sat Oct  8 18:28:02.1255 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
68  Sat Oct  8 18:28:02.1255 2016  7  --- LINE END ---
69  Sat Oct  8 18:28:02.1262 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0905465004
70  Sat Oct  8 18:28:02.1265 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
71  Sat Oct  8 18:28:02.1267 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
72  Sat Oct  8 18:28:02.1269 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
73  Sat Oct  8 18:28:02.1271 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
74  Sat Oct  8 18:28:02.1271 2016  7  Server: SatPI WebServer v0.1
75  Sat Oct  8 18:28:02.1271 2016  7  Location: 
76  Sat Oct  8 18:28:02.1271 2016  7  CSeq: 3534
77  Sat Oct  8 18:28:02.1271 2016  7  cache-control: no-cache
78  Sat Oct  8 18:28:02.1271 2016  7  Content-Type: video/MP2T
79  Sat Oct  8 18:28:02.1271 2016  7  --- LINE END ---
80  Sat Oct  8 18:28:02.1394 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
81  Sat Oct  8 18:28:02.1394 2016  7  CSeq: 3535
82  Sat Oct  8 18:28:02.1394 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
83  Sat Oct  8 18:28:02.1394 2016  7  --- LINE END ---
84  Sat Oct  8 18:28:02.1402 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1194509677
85  Sat Oct  8 18:28:02.1404 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
86  Sat Oct  8 18:28:02.1406 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
87  Sat Oct  8 18:28:02.1408 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
88  Sat Oct  8 18:28:02.1410 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
89  Sat Oct  8 18:28:02.1410 2016  7  Server: SatPI WebServer v0.1
90  Sat Oct  8 18:28:02.1410 2016  7  Location: 
91  Sat Oct  8 18:28:02.1410 2016  7  CSeq: 3535
92  Sat Oct  8 18:28:02.1410 2016  7  cache-control: no-cache
93  Sat Oct  8 18:28:02.1410 2016  7  Content-Type: video/MP2T
94  Sat Oct  8 18:28:02.1410 2016  7  --- LINE END ---
95  Sat Oct  8 18:28:02.1595 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
96  Sat Oct  8 18:28:02.1595 2016  7  CSeq: 3536
97  Sat Oct  8 18:28:02.1595 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
98  Sat Oct  8 18:28:02.1595 2016  7  --- LINE END ---
99  Sat Oct  8 18:28:02.1603 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1615686090
100  Sat Oct  8 18:28:02.1606 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
101  Sat Oct  8 18:28:02.1607 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
102  Sat Oct  8 18:28:02.1609 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
103  Sat Oct  8 18:28:02.1612 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
104  Sat Oct  8 18:28:02.1612 2016  7  Server: SatPI WebServer v0.1
105  Sat Oct  8 18:28:02.1612 2016  7  Location: 
106  Sat Oct  8 18:28:02.1612 2016  7  CSeq: 3536
107  Sat Oct  8 18:28:02.1612 2016  7  cache-control: no-cache
108  Sat Oct  8 18:28:02.1612 2016  7  Content-Type: video/MP2T
109  Sat Oct  8 18:28:02.1612 2016  7  --- LINE END ---
110  Sat Oct  8 18:28:02.1814 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
111  Sat Oct  8 18:28:02.1814 2016  7  CSeq: 3537
112  Sat Oct  8 18:28:02.1814 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
113  Sat Oct  8 18:28:02.1814 2016  7  --- LINE END ---
114  Sat Oct  8 18:28:02.1821 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0385243518
115  Sat Oct  8 18:28:02.1824 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
116  Sat Oct  8 18:28:02.1826 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
117  Sat Oct  8 18:28:02.1827 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
118  Sat Oct  8 18:28:02.1829 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
119  Sat Oct  8 18:28:02.1829 2016  7  Server: SatPI WebServer v0.1
120  Sat Oct  8 18:28:02.1829 2016  7  Location: 
121  Sat Oct  8 18:28:02.1829 2016  7  CSeq: 3537
122  Sat Oct  8 18:28:02.1829 2016  7  cache-control: no-cache
123  Sat Oct  8 18:28:02.1829 2016  7  Content-Type: video/MP2T
124  Sat Oct  8 18:28:02.1829 2016  7  --- LINE END ---
125  Sat Oct  8 18:28:02.2018 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
126  Sat Oct  8 18:28:02.2018 2016  7  CSeq: 3538
127  Sat Oct  8 18:28:02.2018 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
128  Sat Oct  8 18:28:02.2018 2016  7  --- LINE END ---
129  Sat Oct  8 18:28:02.2025 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0587758366
130  Sat Oct  8 18:28:02.2027 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
131  Sat Oct  8 18:28:02.2030 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
132  Sat Oct  8 18:28:02.2031 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
133  Sat Oct  8 18:28:02.2033 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
134  Sat Oct  8 18:28:02.2033 2016  7  Server: SatPI WebServer v0.1
135  Sat Oct  8 18:28:02.2033 2016  7  Location: 
136  Sat Oct  8 18:28:02.2033 2016  7  CSeq: 3538
137  Sat Oct  8 18:28:02.2033 2016  7  cache-control: no-cache
138  Sat Oct  8 18:28:02.2033 2016  7  Content-Type: video/MP2T
139  Sat Oct  8 18:28:02.2033 2016  7  --- LINE END ---
140  Sat Oct  8 18:28:02.2303 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
141  Sat Oct  8 18:28:02.2303 2016  7  CSeq: 3539
142  Sat Oct  8 18:28:02.2303 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
143  Sat Oct  8 18:28:02.2303 2016  7  --- LINE END ---
144  Sat Oct  8 18:28:02.2311 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1109000984
145  Sat Oct  8 18:28:02.2314 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
146  Sat Oct  8 18:28:02.2317 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
147  Sat Oct  8 18:28:02.2318 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
148  Sat Oct  8 18:28:02.2320 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
149  Sat Oct  8 18:28:02.2320 2016  7  Server: SatPI WebServer v0.1
150  Sat Oct  8 18:28:02.2320 2016  7  Location: 
151  Sat Oct  8 18:28:02.2320 2016  7  CSeq: 3539
152  Sat Oct  8 18:28:02.2320 2016  7  cache-control: no-cache
153  Sat Oct  8 18:28:02.2320 2016  7  Content-Type: video/MP2T
154  Sat Oct  8 18:28:02.2320 2016  7  --- LINE END ---
155  Sat Oct  8 18:28:02.2492 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
156  Sat Oct  8 18:28:02.2492 2016  7  CSeq: 3540
157  Sat Oct  8 18:28:02.2492 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
158  Sat Oct  8 18:28:02.2492 2016  7  --- LINE END ---
159  Sat Oct  8 18:28:02.2498 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0698175017
160  Sat Oct  8 18:28:02.2500 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
161  Sat Oct  8 18:28:02.2502 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
162  Sat Oct  8 18:28:02.2504 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
163  Sat Oct  8 18:28:02.2506 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
164  Sat Oct  8 18:28:02.2506 2016  7  Server: SatPI WebServer v0.1
165  Sat Oct  8 18:28:02.2506 2016  7  Location: 
166  Sat Oct  8 18:28:02.2506 2016  7  CSeq: 3540
167  Sat Oct  8 18:28:02.2506 2016  7  cache-control: no-cache
168  Sat Oct  8 18:28:02.2506 2016  7  Content-Type: video/MP2T
169  Sat Oct  8 18:28:02.2506 2016  7  --- LINE END ---
170  Sat Oct  8 18:28:02.2683 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
171  Sat Oct  8 18:28:02.2683 2016  7  CSeq: 3541
172  Sat Oct  8 18:28:02.2683 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
173  Sat Oct  8 18:28:02.2683 2016  7  --- LINE END ---
174  Sat Oct  8 18:28:02.2698 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0173633495
175  Sat Oct  8 18:28:02.2700 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
176  Sat Oct  8 18:28:02.2704 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
177  Sat Oct  8 18:28:02.2707 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
178  Sat Oct  8 18:28:02.2714 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
179  Sat Oct  8 18:28:02.2714 2016  7  Server: SatPI WebServer v0.1
180  Sat Oct  8 18:28:02.2714 2016  7  Location: 
181  Sat Oct  8 18:28:02.2714 2016  7  CSeq: 3541
182  Sat Oct  8 18:28:02.2714 2016  7  cache-control: no-cache
183  Sat Oct  8 18:28:02.2714 2016  7  Content-Type: video/MP2T
184  Sat Oct  8 18:28:02.2714 2016  7  --- LINE END ---
185  Sat Oct  8 18:28:02.2845 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
186  Sat Oct  8 18:28:02.2845 2016  7  CSeq: 3542
187  Sat Oct  8 18:28:02.2845 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
188  Sat Oct  8 18:28:02.2845 2016  7  --- LINE END ---
189  Sat Oct  8 18:28:02.2860 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0249958127
190  Sat Oct  8 18:28:02.2861 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
191  Sat Oct  8 18:28:02.2865 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
192  Sat Oct  8 18:28:02.2869 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
193  Sat Oct  8 18:28:02.2874 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
194  Sat Oct  8 18:28:02.2874 2016  7  Server: SatPI WebServer v0.1
195  Sat Oct  8 18:28:02.2874 2016  7  Location: 
196  Sat Oct  8 18:28:02.2874 2016  7  CSeq: 3542
197  Sat Oct  8 18:28:02.2874 2016  7  cache-control: no-cache
198  Sat Oct  8 18:28:02.2874 2016  7  Content-Type: video/MP2T
199  Sat Oct  8 18:28:02.2874 2016  7  --- LINE END ---
200  Sat Oct  8 18:28:02.3023 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
201  Sat Oct  8 18:28:02.3023 2016  7  CSeq: 3543
202  Sat Oct  8 18:28:02.3023 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
203  Sat Oct  8 18:28:02.3023 2016  7  --- LINE END ---
204  Sat Oct  8 18:28:02.3038 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 2103469422
205  Sat Oct  8 18:28:02.3040 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
206  Sat Oct  8 18:28:02.3044 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
207  Sat Oct  8 18:28:02.3048 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
208  Sat Oct  8 18:28:02.3053 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
209  Sat Oct  8 18:28:02.3053 2016  7  Server: SatPI WebServer v0.1
210  Sat Oct  8 18:28:02.3053 2016  7  Location: 
211  Sat Oct  8 18:28:02.3053 2016  7  CSeq: 3543
212  Sat Oct  8 18:28:02.3053 2016  7  cache-control: no-cache
213  Sat Oct  8 18:28:02.3053 2016  7  Content-Type: video/MP2T
214  Sat Oct  8 18:28:02.3053 2016  7  --- LINE END ---
215  Sat Oct  8 18:28:02.3152 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
216  Sat Oct  8 18:28:02.3152 2016  7  CSeq: 3544
217  Sat Oct  8 18:28:02.3152 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
218  Sat Oct  8 18:28:02.3152 2016  7  --- LINE END ---
219  Sat Oct  8 18:28:02.3167 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0187243392
220  Sat Oct  8 18:28:02.3168 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
221  Sat Oct  8 18:28:02.3172 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
222  Sat Oct  8 18:28:02.3177 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
223  Sat Oct  8 18:28:02.3182 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
224  Sat Oct  8 18:28:02.3182 2016  7  Server: SatPI WebServer v0.1
225  Sat Oct  8 18:28:02.3182 2016  7  Location: 
226  Sat Oct  8 18:28:02.3182 2016  7  CSeq: 3544
227  Sat Oct  8 18:28:02.3182 2016  7  cache-control: no-cache
228  Sat Oct  8 18:28:02.3182 2016  7  Content-Type: video/MP2T
229  Sat Oct  8 18:28:02.3182 2016  7  --- LINE END ---
230  Sat Oct  8 18:28:02.3340 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
231  Sat Oct  8 18:28:02.3340 2016  7  CSeq: 3545
232  Sat Oct  8 18:28:02.3340 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
233  Sat Oct  8 18:28:02.3340 2016  7  --- LINE END ---
234  Sat Oct  8 18:28:02.3355 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1193064728
235  Sat Oct  8 18:28:02.3356 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
236  Sat Oct  8 18:28:02.3361 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
237  Sat Oct  8 18:28:02.3365 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
238  Sat Oct  8 18:28:02.3370 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
239  Sat Oct  8 18:28:02.3370 2016  7  Server: SatPI WebServer v0.1
240  Sat Oct  8 18:28:02.3370 2016  7  Location: 
241  Sat Oct  8 18:28:02.3370 2016  7  CSeq: 3545
242  Sat Oct  8 18:28:02.3370 2016  7  cache-control: no-cache
243  Sat Oct  8 18:28:02.3370 2016  7  Content-Type: video/MP2T
244  Sat Oct  8 18:28:02.3370 2016  7  --- LINE END ---
245  Sat Oct  8 18:28:02.3495 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
246  Sat Oct  8 18:28:02.3495 2016  7  CSeq: 3546
247  Sat Oct  8 18:28:02.3495 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
248  Sat Oct  8 18:28:02.3495 2016  7  --- LINE END ---
249  Sat Oct  8 18:28:02.3510 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1671151159
250  Sat Oct  8 18:28:02.3516 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
251  Sat Oct  8 18:28:02.3517 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
252  Sat Oct  8 18:28:02.3521 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
253  Sat Oct  8 18:28:02.3526 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
254  Sat Oct  8 18:28:02.3526 2016  7  Server: SatPI WebServer v0.1
255  Sat Oct  8 18:28:02.3526 2016  7  Location: 
256  Sat Oct  8 18:28:02.3526 2016  7  CSeq: 3546
257  Sat Oct  8 18:28:02.3526 2016  7  cache-control: no-cache
258  Sat Oct  8 18:28:02.3526 2016  7  Content-Type: video/MP2T
259  Sat Oct  8 18:28:02.3526 2016  7  --- LINE END ---
260  Sat Oct  8 18:28:02.3646 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
261  Sat Oct  8 18:28:02.3646 2016  7  CSeq: 3547
262  Sat Oct  8 18:28:02.3646 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
263  Sat Oct  8 18:28:02.3646 2016  7  --- LINE END ---
264  Sat Oct  8 18:28:02.3661 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1649177200
265  Sat Oct  8 18:28:02.3663 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
266  Sat Oct  8 18:28:02.3667 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
267  Sat Oct  8 18:28:02.3671 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
268  Sat Oct  8 18:28:02.3675 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
269  Sat Oct  8 18:28:02.3675 2016  7  Server: SatPI WebServer v0.1
270  Sat Oct  8 18:28:02.3675 2016  7  Location: 
271  Sat Oct  8 18:28:02.3675 2016  7  CSeq: 3547
272  Sat Oct  8 18:28:02.3675 2016  7  cache-control: no-cache
273  Sat Oct  8 18:28:02.3675 2016  7  Content-Type: video/MP2T
274  Sat Oct  8 18:28:02.3675 2016  7  --- LINE END ---
275  Sat Oct  8 18:28:02.3794 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
276  Sat Oct  8 18:28:02.3794 2016  7  CSeq: 3548
277  Sat Oct  8 18:28:02.3794 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
278  Sat Oct  8 18:28:02.3794 2016  7  --- LINE END ---
279  Sat Oct  8 18:28:02.3808 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0210457853
280  Sat Oct  8 18:28:02.3810 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
281  Sat Oct  8 18:28:02.3816 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
282  Sat Oct  8 18:28:02.3819 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
283  Sat Oct  8 18:28:02.3823 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
284  Sat Oct  8 18:28:02.3823 2016  7  Server: SatPI WebServer v0.1
285  Sat Oct  8 18:28:02.3823 2016  7  Location: 
286  Sat Oct  8 18:28:02.3823 2016  7  CSeq: 3548
287  Sat Oct  8 18:28:02.3823 2016  7  cache-control: no-cache
288  Sat Oct  8 18:28:02.3823 2016  7  Content-Type: video/MP2T
289  Sat Oct  8 18:28:02.3823 2016  7  --- LINE END ---
290  Sat Oct  8 18:28:02.3954 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
291  Sat Oct  8 18:28:02.3954 2016  7  CSeq: 3549
292  Sat Oct  8 18:28:02.3954 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
293  Sat Oct  8 18:28:02.3954 2016  7  --- LINE END ---
294  Sat Oct  8 18:28:02.3959 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0852913812
295  Sat Oct  8 18:28:02.3961 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
296  Sat Oct  8 18:28:02.3962 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
297  Sat Oct  8 18:28:02.3963 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
298  Sat Oct  8 18:28:02.3964 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
299  Sat Oct  8 18:28:02.3964 2016  7  Server: SatPI WebServer v0.1
300  Sat Oct  8 18:28:02.3964 2016  7  Location: 
301  Sat Oct  8 18:28:02.3964 2016  7  CSeq: 3549
302  Sat Oct  8 18:28:02.3964 2016  7  cache-control: no-cache
303  Sat Oct  8 18:28:02.3964 2016  7  Content-Type: video/MP2T
304  Sat Oct  8 18:28:02.3964 2016  7  --- LINE END ---
305  Sat Oct  8 18:28:02.4110 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
306  Sat Oct  8 18:28:02.4110 2016  7  CSeq: 3550
307  Sat Oct  8 18:28:02.4110 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
308  Sat Oct  8 18:28:02.4110 2016  7  --- LINE END ---
309  Sat Oct  8 18:28:02.4118 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1785293194
310  Sat Oct  8 18:28:02.4120 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
311  Sat Oct  8 18:28:02.4122 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
312  Sat Oct  8 18:28:02.4123 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
313  Sat Oct  8 18:28:02.4125 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
314  Sat Oct  8 18:28:02.4125 2016  7  Server: SatPI WebServer v0.1
315  Sat Oct  8 18:28:02.4125 2016  7  Location: 
316  Sat Oct  8 18:28:02.4125 2016  7  CSeq: 3550
317  Sat Oct  8 18:28:02.4125 2016  7  cache-control: no-cache
318  Sat Oct  8 18:28:02.4125 2016  7  Content-Type: video/MP2T
319  Sat Oct  8 18:28:02.4125 2016  7  --- LINE END ---
320  Sat Oct  8 18:28:02.4299 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
321  Sat Oct  8 18:28:02.4299 2016  7  CSeq: 3551
322  Sat Oct  8 18:28:02.4299 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
323  Sat Oct  8 18:28:02.4299 2016  7  --- LINE END ---
324  Sat Oct  8 18:28:02.4306 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1062886709
325  Sat Oct  8 18:28:02.4307 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
326  Sat Oct  8 18:28:02.4309 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
327  Sat Oct  8 18:28:02.4312 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
328  Sat Oct  8 18:28:02.4315 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
329  Sat Oct  8 18:28:02.4315 2016  7  Server: SatPI WebServer v0.1
330  Sat Oct  8 18:28:02.4315 2016  7  Location: 
331  Sat Oct  8 18:28:02.4315 2016  7  CSeq: 3551
332  Sat Oct  8 18:28:02.4315 2016  7  cache-control: no-cache
333  Sat Oct  8 18:28:02.4315 2016  7  Content-Type: video/MP2T
334  Sat Oct  8 18:28:02.4315 2016  7  --- LINE END ---
335  Sat Oct  8 18:28:02.4507 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52121: SETUP rtsp://192.168.30.247:554/?pids=17 RTSP/1.0
336  Sat Oct  8 18:28:02.4507 2016  7  CSeq: 3552
337  Sat Oct  8 18:28:02.4507 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
338  Sat Oct  8 18:28:02.4507 2016  7  --- LINE END ---
339  Sat Oct  8 18:28:02.4515 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0981635709
340  Sat Oct  8 18:28:02.4516 2016  6  [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=UNKNOWN DELSYS
341  Sat Oct  8 18:28:02.4518 2016  6  [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=UNKNOWN DELSYS
342  Sat Oct  8 18:28:02.4520 2016  3  [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
343  Sat Oct  8 18:28:02.4522 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
344  Sat Oct  8 18:28:02.4522 2016  7  Server: SatPI WebServer v0.1
345  Sat Oct  8 18:28:02.4522 2016  7  Location: 
346  Sat Oct  8 18:28:02.4522 2016  7  CSeq: 3552
347  Sat Oct  8 18:28:02.4522 2016  7  cache-control: no-cache
348  Sat Oct  8 18:28:02.4522 2016  7  Content-Type: video/MP2T
349  Sat Oct  8 18:28:02.4522 2016  7  --- LINE END ---
`

**Comment by rezna11** [comment]
it looks like this always happens in case the last transponder fails to lock signal, so the last OK looking message in the log, before it just loops the errors above, is this -  and after that, only satpi restart helps. It is also interesting that locking the transponder fails since it's a channel which I just scanned in using satpi

24  Sat Oct  8 18:38:23.6104 2016  7  [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 52139: SETUP rtsp://192.168.30.247:554/?src=1&freq=11072&pol=v&msys=dvbs&sr=3300&fec=34&pids=none RTSP/1.0
25  Sat Oct  8 18:38:23.6104 2016  7  CSeq: 1
26  Sat Oct  8 18:38:23.6104 2016  7  Transport: RTP/AVP;unicast;client_port=45079-45080
27  Sat Oct  8 18:38:23.6104 2016  7  --- LINE END ---
28  Sat Oct  8 18:38:23.6110 2016  6  [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 1702957685
29  Sat Oct  8 18:38:23.6112 2016  6  [                               src/Stream.cpp:201] Stream: 0, StreamClient[0] with SessionID 1702957685 for dvbs
30  Sat Oct  8 18:38:23.6114 2016  6  [                   src/input/dvb/Frontend.cpp:309] Stream: 0, Parsing transport parameters...
31  Sat Oct  8 18:38:23.6165 2016  7  [                   src/input/dvb/Frontend.cpp:317] Stream: 0, New frequency requested, clearing channel data...
32  Sat Oct  8 18:38:23.6176 2016  7  [                   src/input/dvb/Frontend.cpp:493] Stream: 0, Parsing transport parameters (Finished)
33  Sat Oct  8 18:38:23.6178 2016  7  [                         src/StreamClient.cpp:046] Connection can close: 0
34  Sat Oct  8 18:38:23.6180 2016  7  [                               src/Stream.cpp:258] Stream: 0, Found Streaming type: RTSP
35  Sat Oct  8 18:38:23.6186 2016  6  [                   src/input/dvb/Frontend.cpp:497] Stream: 0, Updating frontend...
36  Sat Oct  8 18:38:23.6189 2016  6  [                   src/input/dvb/Frontend.cpp:858] Stream: 0, Opened /dev/dvb/adapter0/frontend0 fd: 15
37  Sat Oct  8 18:38:23.6191 2016  6  [              src/input/dvb/delivery/DVBS.cpp:105] Stream: 0, Start tuning process for DVB-S(2)...
38  Sat Oct  8 18:38:23.6192 2016  6  [      src/input/dvb/delivery/DiSEqcSwitch.cpp:085] Stream: 0, Sending DiSEqC [e0] [10] [38] [f0]
39  Sat Oct  8 18:38:23.9278 2016  6  [                   src/input/dvb/Frontend.cpp:869] Stream: 0, Waiting on lock...
40  Sat Oct  8 18:38:23.9280 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
41  Sat Oct  8 18:38:24.2048 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
42  Sat Oct  8 18:38:24.3557 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
43  Sat Oct  8 18:38:24.5065 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
44  Sat Oct  8 18:38:24.8068 2016  6  [              src/input/dvb/delivery/DVBS.cpp:105] Stream: 0, Start tuning process for DVB-S(2)...
45  Sat Oct  8 18:38:24.8070 2016  6  [      src/input/dvb/delivery/DiSEqcSwitch.cpp:085] Stream: 0, Sending DiSEqC [e0] [10] [38] [f0]
46  Sat Oct  8 18:38:25.0917 2016  6  [                   src/input/dvb/Frontend.cpp:869] Stream: 0, Waiting on lock...
47  Sat Oct  8 18:38:25.0918 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
48  Sat Oct  8 18:38:25.3422 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
49  Sat Oct  8 18:38:25.4930 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
50  Sat Oct  8 18:38:25.6438 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
51  Sat Oct  8 18:38:25.9442 2016  6  [              src/input/dvb/delivery/DVBS.cpp:105] Stream: 0, Start tuning process for DVB-S(2)...
52  Sat Oct  8 18:38:25.9443 2016  6  [      src/input/dvb/delivery/DiSEqcSwitch.cpp:085] Stream: 0, Sending DiSEqC [e0] [10] [38] [f0]
53  Sat Oct  8 18:38:26.2290 2016  6  [                   src/input/dvb/Frontend.cpp:869] Stream: 0, Waiting on lock...
54  Sat Oct  8 18:38:26.2292 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
55  Sat Oct  8 18:38:26.4822 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
56  Sat Oct  8 18:38:26.6331 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
57  Sat Oct  8 18:38:26.7840 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
58  Sat Oct  8 18:38:27.0843 2016  6  [              src/input/dvb/delivery/DVBS.cpp:105] Stream: 0, Start tuning process for DVB-S(2)...
59  Sat Oct  8 18:38:27.0844 2016  6  [      src/input/dvb/delivery/DiSEqcSwitch.cpp:085] Stream: 0, Sending DiSEqC [e0] [10] [38] [f0]
60  Sat Oct  8 18:38:27.3693 2016  6  [                   src/input/dvb/Frontend.cpp:869] Stream: 0, Waiting on lock...
61  Sat Oct  8 18:38:27.3695 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
62  Sat Oct  8 18:38:27.6223 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
63  Sat Oct  8 18:38:27.7731 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
64  Sat Oct  8 18:38:27.9240 2016  6  [                   src/input/dvb/Frontend.cpp:883] Stream: 0, Not locked yet   (FE status 0x0)...
65  Sat Oct  8 18:38:28.2243 2016  7  [                          src/HttpcServer.cpp:175] RTSP/1.0 200 OK
66  Sat Oct  8 18:38:28.2243 2016  7  CSeq: 1
67  Sat Oct  8 18:38:28.2243 2016  7  Session: 1702957685;timeout=60
68  Sat Oct  8 18:38:28.2243 2016  7  Transport: RTP/AVP;unicast;client_ip=192.168.30.216;client_port=45079-45080
69  Sat Oct  8 18:38:28.2243 2016  7  com.ses.streamID: 0
70  Sat Oct  8 18:38:28.2243 2016  7  --- LINE END ---
71  Sat Oct  8 18:38:28.2251 2016  7  [                     src/socket/TcpSocket.cpp:090] RTSP Client 192.168.30.216:52139 Connection closed with fd: 14
72  Sat Oct  8 18:38:28.2252 2016  6  [                        src/StreamManager.cpp:140] Found SessionID 1702957685 by SocketClient
73  Sat Oct  8 18:38:28.2253 2016  6  [                        src/StreamManager.cpp:158] Found StreamID x - SessionID: 1702957685
74  Sat Oct  8 18:38:28.2254 2016  6  [                               src/Stream.cpp:204] Stream: 0, StreamClient[0] with SessionID 1702957685

**Comment by Barracuda09** [comment]
Hi @rezna11 ,

I can not say anything with these logs.. I like to see the log at the point that it is OK and then fails.

In dmesg you can get the complete log, and please attach the log as txt here.

Thanks again,

Marc

**Comment by rezna11** [comment]
I did a test now, a nice example, but can you explain where is the log file?

**Comment by Barracuda09** [comment]
Yes sorry, it is in syslog

**Comment by Barracuda09** [comment]
you can find it here /var/log/syslog

**Comment by rezna11** [comment]
OK so in my test, which is shown in the attached log, this was done:
1) searched for SAT>IP server on Dreambox, it found SATPI, good
2) went to scan in first frequency - went fine
3) switched to one of the newly found channels - all good
4) then I went to the menu to scan another frequency - nothing, search timed out
5) then I tried to again watch the channel which was previously found - but it didn't work
6) I restarted SATPI and tried to watch the channel again without restarting the Dreambox, it was working 
Log:
[debug.gz](https://github.com/Barracuda09/SATPI/files/518278/debug.gz)

**Comment by Barracuda09** [comment]
Ok... there is something funky going on in this log.

Thanks for this report, I will check it out,

Marc

**Comment by Barracuda09** [comment]
Hi @rezna11 

Could you try and give mea log from first start of SatPI and one tune?

Many thanks,

Marc

**Comment by Barracuda09** [comment]
By the way are you sure that the log is of an DEBUG version??

It seems that there is missing a lot of info which I expect in an debug version.

**Comment by rezna11** [comment]
how do I verify I have the debug version?
by the way the log I posted should include what you're asking - start and tune... or are you looking for something more specific?

**Comment by Barracuda09** [comment]
Yes start of SatPI and one tune (This is not in the log)

You could do:
make clean
make debug

**Comment by rezna11** [comment]
ok then I'm sure it is debug but I copied wrong file earlier, the rigth one added now - the last test starts at Oct  9 20:52:13
[syslog.gz](https://github.com/Barracuda09/SATPI/files/518552/syslog.gz)

**Comment by Barracuda09** [comment]
Which client did you use?

**Comment by rezna11** [comment]
Dreambox 820HD

**Comment by Barracuda09** [comment]
Hi @rezna11 ,

This part looks to me a bit strange from the client:

	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 37965: PLAY rtsp://192.168.30.247:554/stream=0?delpids=20 RTSP/1.0
	Oct  9 20:53:30 raspberrypi SatPI[4071]: CSeq: 73
	Oct  9 20:53:30 raspberrypi SatPI[4071]: Session: 1702957685
	Oct  9 20:53:30 raspberrypi SatPI[4071]: --- LINE END ---
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                        src/StreamManager.cpp:179] Found StreamID 0 - SessionID 1702957685
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                               src/Stream.cpp:204] Stream: 0, StreamClient[0] with SessionID 1702957685
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                   src/input/dvb/Frontend.cpp:309] Stream: 0, Parsing transport parameters...
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                   src/input/dvb/Frontend.cpp:493] Stream: 0, Parsing transport parameters (Finished)
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                         src/StreamClient.cpp:046] Connection can close: 0
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                   src/input/dvb/Frontend.cpp:497] Stream: 0, Updating frontend...
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                   src/input/dvb/Frontend.cpp:925] Stream: 0, Updating PID filters...
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                   src/input/dvb/Frontend.cpp:947] Stream: 0, Remove filter PID: 0020 - fd: 012 - Packet Count: 1
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                   src/input/dvb/Frontend.cpp:517] Stream: 0, Updating frontend (Finished)
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                          src/HttpcServer.cpp:175] RTSP/1.0 200 OK
	Oct  9 20:53:30 raspberrypi SatPI[4071]: RTP-Info: url=rtsp://192.168.30.247/stream=0
	Oct  9 20:53:30 raspberrypi SatPI[4071]: CSeq: 73
	Oct  9 20:53:30 raspberrypi SatPI[4071]: Session: 1702957685
	Oct  9 20:53:30 raspberrypi SatPI[4071]: Range: npt=0.000-
	Oct  9 20:53:30 raspberrypi SatPI[4071]: --- LINE END ---
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                     src/socket/TcpSocket.cpp:090] RTSP Client 192.168.30.216:37965 Connection closed with fd: 10
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                        src/StreamManager.cpp:140] Found SessionID 1702957685 by SocketClient
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                        src/StreamManager.cpp:158] Found StreamID x - SessionID: 1702957685
	Oct  9 20:53:30 raspberrypi SatPI[4071]: [                               src/Stream.cpp:204] Stream: 0, StreamClient[0] with SessionID 1702957685
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                    src/socket/SocketAttr.cpp:126] RTSP Connection from 192.168.30.216 Port 37970 (fd: 10)
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                          src/HttpcServer.cpp:141] RTSP Stream data from client 192.168.30.216 Port 37970: SETUP rtsp://192.168.30.247:554/?src=2&freq=11177&pol=v&msys=dvbs2&sr=13300&fec=56&ro=0.20&mtype=8psk&pids=none RTSP/1.0
	Oct  9 20:53:34 raspberrypi SatPI[4071]: CSeq: 1
	Oct  9 20:53:34 raspberrypi SatPI[4071]: Transport: RTP/AVP;unicast;client_port=46693-46694
	Oct  9 20:53:34 raspberrypi SatPI[4071]: --- LINE END ---
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                        src/StreamManager.cpp:167] Found StreamID x - SessionID x - Creating new SessionID: 0205744672
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                               src/Stream.cpp:190] Stream: 1, Not capable of handling msys=dvbs2
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                               src/Stream.cpp:190] Stream: 2, Not capable of handling msys=dvbs2
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                        src/StreamManager.cpp:195] Found no Stream/Client of interest!
	Oct  9 20:53:34 raspberrypi SatPI[4071]: [                          src/HttpcServer.cpp:175] RTSP/1.0 503 Service Unavailable
	Oct  9 20:53:34 raspberrypi SatPI[4071]: Server: SatPI WebServer v0.1
	Oct  9 20:53:34 raspberrypi SatPI[4071]: Location:
	Oct  9 20:53:34 raspberrypi SatPI[4071]: CSeq: 1
	Oct  9 20:53:34 raspberrypi SatPI[4071]: cache-control: no-cache
	Oct  9 20:53:34 raspberrypi SatPI[4071]: Content-Type: video/MP2T
	Oct  9 20:53:34 raspberrypi SatPI[4071]: --- LINE END ---

Here you can see that the client is closing the connection (port 37965).

Then 4 sec. later a new setup is requested with an new connection (port 37970).
This is assumed to be a new stream request, but because there is only one dvb card it can not find any so it returns 503. But then the client keeps on nagging after this.

The client should have send a teardown request if it is finished to release the resource before closing port 37965.

**Comment by rezna11** [comment]
thanks for looking into this, so it's a client bug - but is it possible to assume each new connection from a client means the previous one should be closed, if there is just one DVB card anyway?

**Comment by Barracuda09** [comment]
Well according to the specs you should close an "session" by giving an TEARDOWN request.
However you can close an TCP connection within on open session, but that does not mean you can close the session.

So what happens here it is that SatPI is waiting for stream 0 to close by a TEARDOWN (Or an time-out occurred). And so the new TCP connection is requesting an SETUP without session id, so it assumes an new session is starting, but does not find the correct resource so it send 503.

Problem: why there is no time-out - This I have to check.

**Comment by Barracuda09** [comment]
Hi @rezna11

Is this still an issue, at this point in time?
If you like and have time could please test this?

Thanks

**Comment by Barracuda09** [comment]
Hi @rezna11 

Is this still an issue, at this point in time?
If you like and have time could please test this?

Thanks

**Comment by Barracuda09** [comment]
I will close this issue, as #56  is also confirmed to be fixed.
If there are any problems, please make a new issue describing the problem.
